modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -2713,6 +2713,37 @@ def obj.class
     end
   end
 
+  describe '#526 without flashes or special issues' do
+    let(:claim_date) { (Time.zone.today - 1.day).to_s }
+    let(:auto_cest_pdf_generation_disabled) { false }
+    let(:data_no_flashes) do
+      temp = File.read(Rails.root.join('modules', 'claims_api', 'spec', 'fixtures',
+                                       'form_526_no_flashes_json_api.json'))
+      temp = JSON.parse(temp)
+      temp['data']['attributes']['autoCestPDFGenerationDisabled'] = auto_cest_pdf_generation_disabled
+      temp['data']['attributes']['claimDate'] = claim_date
+      temp['data']['attributes']['applicationExpirationDate'] = (Time.zone.today + 1.day).to_s
+
+      temp.to_json
+    end
+    let(:path) { '/services/claims/v1/forms/526' }
+    let(:schema) { File.read(Rails.root.join('modules', 'claims_api', 'config', 'schemas', '526.json')) }
+
+    it 'sets the flashes' do
```

Comment:
*nit 
Maybe change test name to `it 'sets the flashes and special_issues'`?
Just because the `describe` says we're dealing with both and the two `expect` statements show we're focused on both.